lookyloo/lookyloo.py

```diff
@@ -761,6 +763,52 @@ def takedown_details(self, hostnode: HostNode) -> dict[str, Any]:
         to_return['all_emails'] = list(to_return['all_emails'])
         return to_return
 
+    def takedown_filtered(self, hostnode: HostNode) -> dict[str, Any] | None:
+        config = configparser.ConfigParser()
+        config.optionxform = str
+        config.read('/home/amaraj/Stage/Workshop/domain.ini')
```

Comment:
In https://github.com/AntoniaBK/lookyloo/blob/e45b7c4346706160d21e0e27048c26c03128277a/lookyloo/default/helpers.py#L20 is the function get_homedir() that can be used to write the path more generel
I guess it should be the following (if the file is in config/domain.ini)
```diff
- config.read('/home/amaraj/Stage/Workshop/domain.ini')
+ ignorelist_path = get_homedir() / 'config' / 'domain.ini'
+ config.read(ignorelist_path)
```

Reply:
Thanks, just changed it!